Release notes for 1.17.6 (#13883)
istio/istio.io: content/en/news/releases/1.17.x/announcing-1.17.6/index.md

---
title: Announcing Istio 1.17.6
linktitle: 1.17.6
subtitle: Patch Release
description: Istio 1.17.6 patch release.
publishdate: 2023-09-19
release: 1.17.6
---

This release contains bug fixes to improve robustness. This release note describes what is different between Istio 1.17.5 and Istio 1.17.6.

{{< relnote >}}

## Changes

> - **Fixed** SELinux issue on CentOS9/RHEL9 where iptables-restore isn't allowed to open files in `/tmp`. Rules passed to iptables-restore are no longer written to a file, but are passed via stdin. ([Issue #42485](https://github.com/istio/istio/issues/42485))

Reviewer: Should be spaces between OS and the number. Also can remove from .spelling
```suggestion
- **Fixed** SELinux issue on CentOS 9/RHEL 9 where iptables-restore isn't allowed to open files in `/tmp`. Rules passed to iptables-restore are no longer written to a file, but are passed via stdin. ([Issue #42485](https://github.com/istio/istio/issues/42485))
```

> - **Fixed** an issue where Istio should be using IMDSv2 as possible on AWS. ([Issue #45825](https://github.com/istio/istio/issues/45825))

Reviewer: Can remove from spelling since backticks?
```suggestion
- **Fixed** an issue that Istio should prefer `IMDSv2` on AWS. ([Issue #45825](https://github.com/istio/istio/issues/45825))
```

> - **Fixed** `meshConfig.defaultConfig.sampling` is ignored when there's only default providers. ([Issue #46653](https://github.com/istio/istio/issues/46653))

Reviewer: ```suggestion
- **Fixed** an issue where `meshConfig.defaultConfig.sampling` is ignored when there are only default providers. ([Issue #46653](https://github.com/istio/istio/issues/46653))
```
